### ISSUE TITLE
OVH plugin using DnsAlias fails if not using subdomain of the OVHdomain

### ISSUE THREAD

**Issue by ctdml** [rmbolger/Posh-ACME#535]
To me it apperas that the TXT chalange entries cant be added to the OVHdomain directly, but must be added to a subdomain of the OVHdomain (so same zone file) which is the workaround for me. The below error gets thrown if not using a subdomain as DnsAlias. My understanding is that in this situation $recShort beeing empty is correct, but the function cant handle this.

Thanks a lot for providing Posh-ACME and its DNS plugins!
Best Regards

```
Submit-ChallengeValidation -verbose
AUSFÜHRLICH: Publishing challenge for Domain one.company.eu with Token
XXXXXXXXXXXXXXXXXXXXXXXXXX using Plugin OVH and DnsAlias 'company.provider.eu'.
AUSFÜHRLICH: CTEDML, first command in Add-DnsTXT in Plugin
AUSFÜHRLICH: CTEDML, next is find-ovhdomain in Plugin for company.provider.eu
AUSFÜHRLICH: CTEDML, OVHDomain is company.provider.eu
AUSFÜHRLICH: CTEDML, recShort is
Submit-ChallengeValidation : Das Argument kann nicht an den Parameter "RecordShort" gebunden werden, da es sich um
eine leere Zeichenfolge handelt.
In Zeile:1 Zeichen:1
+ Submit-ChallengeValidation -verbose
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidData: (:) [Submit-ChallengeValidation], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Submit-ChallengeValidation   
```

**Comment by rmbolger** [comment]
Hi @ctdml. So sorry for the delayed response on this. You are entirely correct. This is a bug with an internal function not allowing for an empty subdomain. I'm in the process of pushing a fix right now.

**Comment by rmbolger** [comment]
Fix is live in 4.21.0
